docs/querying-data.rst

```diff
@@ -0,0 +1,207 @@
+Querying data in Kingfisher Views
+=================================
+
+Summary tables in Kingfisher Views are generated from data in `Kingfisher Process <https://kingfisher-process.readthedocs.io/en/latest/>`__, which is organized by collection. Each collection contains either data downloaded from an OCDS publisher using a `Kingfisher Scrape <https://kingfisher-scrape.readthedocs.io/en/latest/>`__ spider, or loaded locally.
+
+A Kingfisher Views schema can contain summaries of one or more collections.
+
+OCDS data can be published using either `releases or records <https://standard.open-contracting.org/latest/en/getting_started/releases_and_records/>`__. Releases are point in time updates about a contracting process and a record provides an index of all the releases for a contracting process.
+
+Kingfisher Process automatically transforms OCDS releases to create compiled release collections,  containing a single compiled release per contracting process populated with the latest value of each field.
+
+Tables in Kingfisher Views can contain summaries of:
+
+* individual releases
+* compiled releases generated by Kingfisher Process
+* compiled releases published as part of an OCDS record
+
+Since most analysis is much easier to perform on compiled releases, we recommend working with compiled releases to begin with.
```

Comment:
This should be worked into either the main page of other relevant parts of the documentation. We shouldn't re-explain Kingfisher Views in multiple locations.

Reply:
In general I agree we shouldn't explain things in multiple places.

The original idea of this documentation page was to have a single quick start guide to share with users when the helpdesk grants them access to OCP's hosted instance of Kingfisher.

My thinking is that most of these users don't want to know 'how does Kingfisher views work' but 'how do I query OCDS data' and this introduction is intended to be the bare minimum of information a user needs to know in order to do that.

If we just integrate the documentation elsewhere and link to it from here my concern is that we end up with lots of links that users don't follow, instead of a one stop shop to get someone started.

Reply:
Hmm, if this is intended for external Redash users, then we have a lot more knowledge building to do. We'd need a better description of data provenance (data sources, original formats, transforms, summaries) – so that users understand which collection or view contains the data they want to query. We'd want to link back to Kingfisher Process' querying data page.

I recommend re-thinking this page as a useful set of example queries, that serves internal users, external Kingfisher users, and external Redash users. For Redash users specifically, we can design a new page that does more of that preamble work, and that links to this page for specific queries/operations.

Reply:
Also, I don't know of any evidence that users don't follow links. Links are what make the web.

Reply:
I've reviewed this content and I think it's sufficiently covered by the home page and other content linked from there, so I've removed it in https://github.com/open-contracting/kingfisher-views/pull/109/commits/e9666e4517a8ef2b8f56e53da6b6bef795c67a8b.